api/repositories/search.repository.js

```diff
@@ -0,0 +1,64 @@
+import prisma from "../prisma/prisma.js";
+export const getProductsAndStores = async (query) => {
+    const stores = await prisma.stores.findMany({
+        where: {
+            OR: [
+            {
+              name: {
+                contains: query,
+              },
+            },
+            { 
+              description: { 
+                contains: query 
+            } 
+            },
+          ],
+        }
+    })
+    const products = await prisma.products.findMany({
+        where: {
+            OR: [
+            {
+              name: {
+                contains: query,
+              },
+            },
+            { 
+              description: { 
+                contains: query 
+            } 
+            },
+          ],
+        }
+    })
+    return {stores, products};
+}
+export const getSearchQueryDetails = async (id) => {
```

Comment:
is this just not findByStoreId(..)?

Reply:
It is findStoreByStoreId() or findProductById() depends on the e Id. If id is storeId give me this store but If the id is ProductId give me the Product